### PR TITLE
3160: Update confirmation page content on org model view [meoward]

### LINKED ISSUE #3160: Request confirmation: Update language on this page
### Issue description

Within the org model, we need to slightly modify the text that appears on this confirmation page to match what's in [this document](https://docs.google.com/document/d/1GTTeRvKfe3lA_w2U4X_CBk8Q7BpMe7W0m_eVZsTHzEE/edit?tab=t.qzpcx388pip#heading=h.31pyx7pcr6s3). That includes removing the bullets and replacing them with a sentence that reads: 

> During our review we’ll verify that your requested domain meets our naming requirements.

### Acceptance criteria

- [x] For org model only - update the content on that page as described above.

### Additional context

_No response_

### Links to other issues

_No response_

### PR DESCRIPTION
**Pull request by erinysong**
## Ticket

Resolves #3160 

## Changes

- Updates domain request submit confirmation page content to reflect changes requested in ticket's [reference doc](https://docs.google.com/document/d/1GTTeRvKfe3lA_w2U4X_CBk8Q7BpMe7W0m_eVZsTHzEE/edit?tab=t.qzpcx388pip).

## Setup

On meoward, submit a domain request with the org model flag off then on. Verify the text on the submit confirmation page matches the content requested in the ticket.

**With org model flag off**
image

**With org model flag on**
image

#### Satisfied acceptance criteria and met development standards

- [x] Met the acceptance criteria, or will meet them in a subsequent PR
- [x] Created/modified automated tests
- [x] Update documentation in READMEs and/or onboarding guide

#### Ensured code standards are met (Original Developer)

- [x] If any updated dependencies on Pipfile, also update dependencies in requirements.txt. - NA
- [x] Interactions with external systems are wrapped in try/except - NA
- [x] Error handling exists for unusual or missing values - NA

#### Validated user-facing changes (if applicable)

- [ ] Tag @dotgov-designers in this PR's Reviewers for design review. If code is not user-facing, delete design reviewer checklist
- [ ] Verify new pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Verified code meets all checks above. Address any checks that are not satisfied
- [ ] Reviewed this code and left comments. Indicate if comments must be addressed before code is merged
- [ ] Checked that all code is adequately covered by tests
- [ ] Verify migrations are valid and do not conflict with existing migrations

#### Validated user-facing changes as a developer
**Note:** Multiple code reviewers can share the checklists above, a second reviewer should not make a duplicate checklist. All checks should be checked before approving, even those labeled N/A. 

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] (Rarely needed) Tested as both an analyst and applicant user

#### Verified that the changes match the design intention

- [ ] Checked that the design translated visually
- [ ] Checked behavior. Comment any found issues or broken flows.
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Tested with multiple browsers (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari
- [ ] (Rarely needed) Tested as both an analyst and applicant user

### References
- [Code review best practices](../docs/dev-practices/code_review.md)

### CODE REVIEW

**Review by therealslimhsiehdy — APPROVED** [review]
LGTM:

**Org model flag off:**
Screenshot 2024-12-11 at 3 58 44 PM

**Org model flag on:**
Screenshot 2024-12-11 at 4 01 48 PM

File: src/registrar/templates/domain_request_done.html [hunk 1]
```diff
@@ -26,13 +26,17 @@ <h1>Thanks for your domain request!</h1>
...
-    
-  <p>During our review we’ll verify that:</p> 
-  <ul class="usa-list">
-    <li>Your organization is eligible for a .gov domain.</li>
-    <li>You work at the organization and/or can make requests on its behalf.</li> 
-    <li>Your requested domain meets our naming requirements.</li>
-  </ul>
+  
+  {% if has_organization_feature_flag %}
+  <p>During our review we’ll verify that your requested domain meets our naming requirements.</p>
```
Comment (Katherine-Osos) [inline src/registrar/templates/domain_request_done.html:31]:
```suggestion
  <p>During our review, we’ll verify that your requested domain meets our naming requirements.</p> 
```

File: src/registrar/templates/domain_request_done.html [hunk 2]
```diff
@@ -26,13 +26,17 @@ <h1>Thanks for your domain request!</h1>
...
-  <ul class="usa-list">
-    <li>Your organization is eligible for a .gov domain.</li>
-    <li>You work at the organization and/or can make requests on its behalf.</li> 
-    <li>Your requested domain meets our naming requirements.</li>
-  </ul>
+  
+  {% if has_organization_feature_flag %}
+  <p>During our review we’ll verify that your requested domain meets our naming requirements.</p> 
+  {% else %}
+    <p>During our review we’ll verify that:</p>
```
Comment (Katherine-Osos) [inline src/registrar/templates/domain_request_done.html:33]:
```suggestion
    <p>During our review, we’ll verify that:</p> 
```

**Comment by Katherine-Osos** [comment]
@erinysong I realized we were missing a couple commas, so I added those as suggested code edits. I also updated the Google doc.

**Comment by erinysong** [comment]
thank you for the update @Katherine-Osos! Just added those in 😄

**Pull request merged**